app/controllers/recipes_controller.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+class RecipesController < ApplicationController
+  load_and_authorize_resource
+
+  def index
+    @recipes = current_user.recipes.all
+  end
+
+  def show
+    @recipe = Recipe.includes(:recipe_foods).find_by_id(params[:id])
```

Comment:
- [ ] Kindly implement the `Inventory shopping list` as per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/recipe-app/buisness_requirements.md#:~:text=Inventory%20shopping%20list%3A,of%20the%20missing%20food.).